website/blog/deployers.md

```diff
@@ -347,14 +368,28 @@ here. You can also review the implementations of our [`weaver
 multi`][weaver_github] and [`weaver gke`][weaver_gke_github] deployers as
 reference.
 
+- **Longevity and Persistence**. The multiprocess deployer in the previous
+  section lived only as long as the application it deployed. If the deployer
+  failed, it failed the application along with it. Real-world deployers should
```

Comment:
Is it true that the application would fail alongwith the deployer implemented in this post?  I'd think that the child processes would continue to linger around and reparented to init process.  I am not suggesting any changes to this sentence (it conveys what's needed) but I am asking this out of curiosity - e.g. if weavelets take any actions such as killing themselves on loosing contact with the deployer to tear down the entire application?

Reply:
I removed the sentence because I realized it wasn't needed, so I didn't want to cause any confusion. If the deployer dies, I think the pipes to the weavelets will close and they will crash.